
package game;

message CharacterPacket {
  required uint64 account = 1;
  required string name = 2;
  required int32 level = 3;
  required int32 exp = 4;
  required int32 gem = 5;
  required int32 coin = 6;

}

